### PR TITLE
Language: Add portuguese translation

### PR DESCRIPTION
**Pull request by ceifa**

### CODE REVIEW

File: lua/terrortown/lang/pt.lua [hunk 1]
```diff
@@ -0,0 +1,1422 @@
+-- Portuguese language strings
+
+local L = LANG.CreateLanguage("Português (Brasil)")
```
Comment (TimGoll) [inline lua/terrortown/lang/pt.lua:3]: I know this is WIP, but this should be `local L = LANG.CreateLanguage("pt")`

Comment (TimGoll) [reply lua/terrortown/lang/pt.lua:3]: also if we differentiate between "normal" portuguese and brasilian portuguese, we should rerflect that in their identifiers. As someone that doesn't speak this language I have to ask if this differentiation is necessary.

Comment (Histalek) [reply lua/terrortown/lang/pt.lua:3]: Language codes/identifiers are a mess and our usage doesn't help either :D

But following our current scheme this should be `pt_br`. This also allows for an european portuguese translation in the future.

```suggestion
local L = LANG.CreateLanguage("pt_br")
```

Comment (Histalek) [reply lua/terrortown/lang/pt.lua:3]: Furthermore the filename should be changed accordingly e.g.: `pt_br.lua`

Comment (ceifa) [reply lua/terrortown/lang/pt.lua]: I changed it to `pt_br`. Brazilian portuguese it's a little bit different of Portugal portuguese.

**Review by Histalek — APPROVED** [review]
Thank you for your work!

**Review by TimGoll — APPROVED** [review]

**Pull request merged**